src/command/output.rs

```diff
@@ -36,6 +39,28 @@ impl RoverStdout {
                 }
                 println!("{}", &hash);
             }
+            RoverStdout::SubgraphList(details) => {
+                if atty::is(Stream::Stdout) {
+                    tracing::info!("Subgraphs:");
+                }
+
+                let mut table = Table::new();
+                table.add_row(row!["Name", "Routing Url", "Last Updated"]);
+
+                for subgraph in &details.subgraphs {
+                    table.add_row(row![
+                        subgraph.name,
+                        subgraph.url.clone().unwrap_or_else(|| "".to_string()),
```

Comment:
```suggestion
                        subgraph.url.clone().unwrap_or_else(|| "N/A".to_string()),
```

^ it feels a bit weird for this to be completely empty in the table

Reply:
That makes sense! I can also replace `""` with `N/A` since empty string also doesn't technically have anything lol